cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/documentstore/LocalDocument.java

```diff
@@ -15,7 +15,7 @@
 package com.cloudant.sync.documentstore;
 
 /**
- * A local Document, this document does not have a history, or the concept of revisions
+ * A local Document. {@code LocalDocument}s do not have a history, or the concept of revisions
```

Comment:
I'm in two minds about that, Android Studio gives a warning if documentation links to itself, but the advantage of using a link is that it would catch any typos in the link name